docs/pt-br/README.md

```diff
@@ -0,0 +1,42 @@
+
+<!-- markdown-link-check-disable -->
+# Documentação para o Cosmos Hub
+
+Bem-vindo à documentação do **aplicativo para o Cosmos Hub: `gaia`**
+
+## O que é Gaia?
+
+- [Introdução ao software `gaia`](./gaia-tutorials/what-is-gaia.md)
+
+## Junte-se à rede principal do Cosmos Hub
+
+- [Instalar o aplicativo `gaia`](./gaia-tutorials/installation.md)
+
+- [Prepare um nó completo e junte-se a uma rede principal](./gaia-tutorials/join-mainnet.md)
+
+- [Atualização para ser um nó validador](./validators/validator-setup.md)
+
+## Junte-se à rede de teste pública do hub Cosmos
+
+- [Join testnet](./hub-tutorials/join-testnet.md)
+
+## Prepare seu próprio Testnet `gaia`
+
+- [Prepare seu próprio Testnet `gaia`](../hub-tutorials/deploy-testnet.md)
```

Comment:
I usually see people use feminine articles for testnets, specially since the translation "rede de teste" is feminine.

```suggestion
## Prepare sua própria Testnet `gaia`

- [Prepare sua própria Testnet `gaia`](../hub-tutorials/deploy-testnet.md)
```